### ISSUE TITLE
Table does refresh when perPage changed and displaying various amount of rows

### ISSUE THREAD

**Issue by R00118189** [rappasoft/laravel-livewire-tables#259]
noticed it when upgraded from 1.2.2 to 1.4.0

It is weird behaviour and the best to illustrate with the video.
![LivewireTablesPagination](https://user-images.githubusercontent.com/6721357/116782525-898aa500-aa81-11eb-8843-18b123ad4fc6.gif)
You can notice some of the rows are not updated some are added to DOM maybe it is a livewire recent update and not the package or package custom pagination classes. 

Edit: 
I created simple Livewire component used withPagination and it worked as intended. So I suspect something is within the package which renders the table in a weird way.

**Comment by rappasoft** [comment]
If you have published the views, delete them and see if it makes a difference. Otherwise drop your component code here.

**Comment by R00118189** [comment]
Deleted vendor file in resources as well as a main vendor then `composer install` and  run both tailwind and bootstrap "themes" still the same result, component as basic as it can be

```php
<?php

namespace App\Http\Livewire;

use App\Models\Item;
use Illuminate\Database\Eloquent\Builder;
use Illuminate\Support\Str;
use Rappasoft\LaravelLivewireTables\DataTableComponent;
use Rappasoft\LaravelLivewireTables\Views\Column;

class Index extends DataTableComponent
{
    public function query() : Builder
    {
        return Item::query();
    }

    public function columns() : array
    {
        return [
            Column::make('Column 1', 'column_1')->format(fn ($value) => Str::limit($value, 15)),
            Column::make('Column 2', 'column_2'),
            Column::make('Column 3', 'column_3'),
            Column::make('Column 4', 'column_4'),
        ];
    }
}

```
![vendordeleted](https://user-images.githubusercontent.com/6721357/116789776-7c34e100-aaa8-11eb-9f5e-36b83eeab53f.gif)

**Comment by rappasoft** [comment]
Try clearing the session or trying in incognito mode and see if it makes a difference.

**Comment by R00118189** [comment]
All done as instructed used checked chrome, safari and firefox incognito modes problem persists. I just want to add that if searching the count is correct but the table renders some residual data.  it almost looks like this part causes some form of view "caching" 

```php
   @forelse ($rows as $index => $row)
       <x-livewire-tables::table.row
         ...
      </x-livewire-tables::table.row>
   @empty
```

if component `<x-livewire-tables::table.row>` is replaced with pure data `$row` 

```php
   @forelse ($rows as $index => $row)
      {{ $row }}
   @empty
```
it renders fine and is not "caching" the view

**Comment by rappasoft** [comment]
Hmm ok. Give me everything you can in order to recreate this, whether it be the full project if possible. Otherwise the whole component, model, and maybe a seeder. I'll figure it out.

**Comment by R00118189** [comment]
Yes it was the UUID on the model. I did not add my uuids trait to this model. so autoincrement was not set to false and key to string

```php
 public function getIncrementing()
    {
        return false;
    }

public function getKeyType()
    {
        return 'string';
    }
```
`id` field was returning 0 or if uuid value was starting with digits only those starting digits were returned. I only found out when printed {{$row}} in the loop. so the id field was messing with the loop.
 
apologies I think we can close this issue. but if you wish to recreate this use model with uuid but without the `trait`
